backend/editor/entries.py

```diff
@@ -438,6 +439,28 @@ async def get_children(self, entry):
         result = await get_current_transaction().run(query, {"id": entry})
         return await async_list(result)
 
+    async def get_stopwords_dict(self):
+        """
+        Helper function used for getting all stopwords in a taxonomy, in the form of a dictionary
+        where the keys are the language codes, and the values are the stopwords in the
+        corresponding language
+        """
+        query = f"""
+            MATCH (s:{self.project_name}:STOPWORDS)
+            WITH keys(s) AS properties, s
+            UNWIND properties AS property
+            WITH s, property
+            WHERE property STARTS WITH 'tags_ids'
+            RETURN property AS tags_ids_lc, s[property] AS stopwords
+        """
+        result = await get_current_transaction().run(query)
+        records = await async_list(result)
+        stopwords_dict = {}
+        for record in records:
+            language_code = record["tags_ids_lc"].split("_")[-1]
+            stopwords_dict[language_code] = record["stopwords"]
+        return stopwords_dict
```

Comment:
nit: I understand if you don't want to do this, but it would be cleaner and more readable to create a Pydantic model for the return type
(The best would be to have a simple Neo4J query and to have the data transformation step as part of a Pydantic model validator)

Reply:
sorry I think I'll pass on that one 😅